README.md

```diff
@@ -2,13 +2,16 @@
 
 # Capstone project for module two
 
-In this project, I set up my To-Do List web page. My goal was add the functionality to add and display each todo task inside my task list. 
+In this project, we set up our Capstone project of dynamic web page. Our goal is implement 
+the all lessons we learned in javascript module. 
 
-![screenshot](/public/Screenshot%20from%202022-10-06%2009-06-02.png)
+![screenshot](./public/Screenshot%20from%202022-10-20%2011-48-34.png)
 
 ## :red_circle: Live Demo
 You can view my hosted website in the link shown below.
-GitHub: [@JUPAC](https://juaryimami.github.io/capestone-module-two/dist/)
+
+- Deployed: [Live webpage](https://juaryimami.github.io/capstone-module-two/dist/)
+- Video: [Loom video link](https://www.loom.com/share/05024db5a0264262a79cae35baee1cd1)
```

Comment:
In this loom video recording, Only one group member appeared and explained the project. Whereas there were two group members who contributed to this project. 

Please record the video again in which all group members participate and explain the project and their favorite piece of code. 👍🏻

Reply:
Thank you. We will make the change. However could you suggest any software besides loom and zoom (since it kept crashing in our attempts to record)